### ISSUE TITLE
Duplicate releases displayed (add arch differentiation on MacOS)

### ISSUE THREAD

**Issue by DanMossa** [fluttertools/sidekick#246]
I'm running this on an m1 mac mini.

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/10294777/176020181-308edb3b-7280-4643-a054-54d07b77ded4.png">

**Comment by aguilaair** [comment]
What's the issue here? Are there's versions not installed?

**Comment by DanMossa** [comment]
Sorry, by multiple I meant duplicate. Every version has a duplicated row

**Comment by karniv00l** [comment]
I'm seeing the same

<img width="328" alt="Screenshot 2022-07-01 at 22 34 27" src="https://user-images.githubusercontent.com/3144291/176964667-51cc1251-d1fe-420f-af57-d42c03a7714c.png">

**Comment by aguilaair** [comment]
> I'm seeing the same
> 
> 
> 
> <img width="328" alt="Screenshot 2022-07-01 at 22 34 27" src="https://user-images.githubusercontent.com/3144291/176964667-51cc1251-d1fe-420f-af57-d42c03a7714c.png">
> 
> 

Is this also in an M1 Mac?

**Comment by karniv00l** [comment]
Ah sorry - yes

```
MacBook Pro 16, Apple M1 Max
macOS Monterey 12.4
```

Any way I can help you debugging this?

**Comment by aguilaair** [comment]
I don't have a Mac, so if you could clone the project, figure out what's happening and push a PR that'd definitely help 😂😜

Otherwise we'll have to wait on @leoafarias to be able to take a look :)

**Comment by karniv00l** [comment]
Ok, got it.

Since Flutter is being built for both `x64` and `arm64` platforms, we have more versions to choose from.
See releases.json: https://raw.githubusercontent.com/fluttertools/fvm/main/releases_macos.json

This would require some work to indicate architectures in both the `Sidekick UI` and `fvm cli`.

```bash
fvm releases

...
Apr 13 22  │ 2.13.0-0.1.pre
Apr 13 22  │ 2.13.0-0.1.pre
Apr 18 22  │ 2.10.5
Apr 20 22  │ 2.13.0-0.2.pre
Apr 20 22  │ 2.13.0-0.2.pre
Apr 27 22  │ 2.13.0-0.3.pre
Apr 27 22  │ 2.13.0-0.3.pre
May 5 22   │ 2.13.0-0.4.pre
May 5 22   │ 2.13.0-0.4.pre
May 11 22  │ 3.0.0
May 11 22  │ 3.0.0
May 20 22  │ 3.0.1
May 20 22  │ 3.0.1
May 26 22  │ 3.1.0
May 26 22  │ 3.1.0
Jun 10 22  │ 3.0.2
Jun 10 22  │ 3.0.2
Jun 15 22  │ 3.1.0-9.0.pre
--------------------------------------
Jun 15 22  │ 3.1.0-9.0.pre     beta
--------------------------------------
Jun 22 22  │ 3.0.3
Jun 22 22  │ 3.0.3
Jul 1 22   │ 3.0.4
--------------------------------------
Jul 1 22   │ 3.0.4             stable
--------------------------------------
```

**Comment by karniv00l** [comment]
FVM issue: https://github.com/fluttertools/fvm/issues/437

**Comment by aguilaair** [comment]
Partial solution comitted, still waiting on arch-based solution, must be provided through FVM, @leoafarias

**Comment by aguilaair** [comment]
Available in 1.1.0
